src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -98,7 +98,7 @@ public function getVisibility($path)
      *
      * @param  string  $path
      * @param  string  $visibility
-     * @return void
+     * @return bool
      */
     public function setVisibility($path, $visibility)
```

Comment:
Contract `@return` [differs](https://github.com/laravel/framework/blob/5.1/src/Illuminate/Contracts/Filesystem/Filesystem.php#L62), but should stay `void`, I think.